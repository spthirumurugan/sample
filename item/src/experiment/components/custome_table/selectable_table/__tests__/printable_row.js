import React from 'react';
import renderer from 'react-test-renderer';
import PrintableRow from '../printable_row';

describe('components', () => {
  describe('<PrintableRowu />', () => {
    it('should render', () => {
      const row = renderer.create(
        <PrintableRow cells={['prop1', 'prop2']} uIds={['1', '2', '3', '4']} />
      ).toJSON();
      expect(row).toMatchSnapshot();
    });
  });
});
