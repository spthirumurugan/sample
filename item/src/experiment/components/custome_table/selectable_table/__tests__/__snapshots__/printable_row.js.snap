// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components <PrintableRowu /> should render 1`] = `
<tr>
  <td>
    prop1
  </td>
  <td>
    prop2
  </td>
</tr>
`;
